electrumsv/network.py

```diff
@@ -58,6 +58,15 @@
 MAX_CONCEIVABLE_REORG_DEPTH = 500
 
 
+async def header_sync_state_middleware(wallet: Wallet) -> bool:
```

Comment:
This function is superflous, the JSON RPC logic should just call the function on the network.